contracts/staking/interfaces/IValidatorManager.sol

```diff
@@ -39,6 +38,19 @@ struct ValidatorManagerSettings {
     uint8 maximumChurnPercentage;
 }
 
+struct SubnetConversionData {
+    bytes32 convertSubnetTxID;
+    bytes32 blockchainID;
+    address validatorManagerAddress;
+    InitialValidator[] initialValidators;
+}
+
+struct InitialValidator {
```

Comment:
I think we should have the name here match the spec. The spec [update](https://github.com/avalanche-foundation/ACPs/pull/146/files) calls this `ValidatorData`. Between the two, I prefer `InitialValidator`, unless there's another scenario in which `ValidatorData` would be used outside of initialization. @michaelkaplan13